apps/api/tests/test_webhooks.py

```
@@ -139,6 +142,193 @@ def test_unrecognized_event_type_returns_200_without_side_effects(webhook_client
+        # effect for the rest of the test almost by accident). Real inserts need the same
+        # explicit tenant context the router itself sets before writing a row.
+        set_tenant_session_context(setup, org.tenant_id, 0)
+        # A direct insert, not installation_repo.create()/upsert() -- that helper's own
+        # db.commit() + db.refresh() immediately after can itself hit issue #330's documented
+        # nested-commit/RLS-timing flakiness under genuine thread concurrency (same class as
+        # test_org_membership_repo.py's xfail'd tests), which is no part of what this test
+        # means to exercise. Nothing after this needs the row's server-generated defaults.
+        setup.add(
+            GitHubInstallation(
+                account_login="acme-lock-test", account_type="Organization", auth_mode="app",
+                installation_id=9001, token_ref="tok_acme-lock-test", org_id=org.id, tenant_id=org.tenant_id,
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to argument: "token_ref"

(S106)